### ISSUE TITLE
The usage of TransferCompound device

### ISSUE THREAD

**Issue by chaeunl** [IBM/aihwkit#91]
## Description
In the docs, I think it is not clear which device is "transferred" or "transferring" Following the paper [1], there are two types of array: A for accumulation of gradients and C for weights of neural network. When defining devices by the parameters, `unit_cell_devices`, we have no idea about which index of the device corresponds to A and C. As I looked over the C++ source code, 0-th index of `unit_cell_devices` corresponds to A and 1-st index to C.  

## Expected behavior
I hope you to describe the details at the usage of `unit_cell_device`.

## Other information
[1] Tayfun Gokmen and Wilfried Hanesch, Algorithm for Training Neural Networks on Resistive Device Arrays, Frontiers in Neuroscience, 2020.

* Pytorch version: 1.8.0
* Package version:
* OS: Ubuntu 18.04.2
* Python version: 3.8
* Conda version (or N/A):

**Comment by maljoras** [comment]
Yes, indeed, the first device is always the one receiving that SGD updates and the transfer is done from lower indexed devices to higher.

**Comment by diego-plan9** [comment]
Thanks Malte - @chaeunl, do you think we can close the issue after the updates at #92?

**Comment by chaeunl** [comment]
@diego-plan9 and @maljoras , of course. The description is good enough! Thank you for revising the docs :)

**Comment by diego-plan9** [comment]
Wonderful! Thanks to you for pointing it out :+1:
